### PR TITLE
Fix crash on iOS7 due nil value of durationValue

### PR DESCRIPTION
**Pull request by Takeno**
Refer #15

### CODE REVIEW

**Comment by johanneslumpe** [comment]
Thanks for the PR, I will take a look as soon as I can, a bit busy at the moment. :)

File: RNKeyboardEventsManager.m [hunk 1]
```diff
@@ -73,6 +73,11 @@ - (NSDictionary*) makeBodyFromNotification:(NSNotification*)notification {
   CGRect keyboardEndRect = [[[notification userInfo] objectForKey:UIKeyboardFrameEndUserInfoKey] CGRectValue];
   NSNumber *durationValue = [[notification userInfo] objectForKey:UIKeyboardAnimationDurationUserInfoKey];
 
+    
+  // Normalization: if durationValue is null, set it to 0
+  if(durationValue == nil)
+    durationValue = [[NSNumber alloc] initWithDouble:0];
+
```
Comment (johanneslumpe) [inline RNKeyboardEventsManager.m:81]: Could you update the code to use curly braces please and use the convenience method, like this:

```
if (durationValue == nil) {
  durationValue = [NSNumber numberWithDouble:0];
}
```

@Takeno

**Comment by Takeno** [comment]
- NSNumber initialization changed
- Adjusted according the Coding Style

**Comment by Takeno** [comment]
Done :+1:

**Comment by johanneslumpe** [comment]
Perfect!

**Comment by johanneslumpe** [comment]
Just published as `0.4.5` :+1:

**Pull request merged**